docs/book/src/user/quick-start.md

```diff
@@ -565,7 +565,7 @@ The Docker provider is not designed for production use and is intended for devel
 
 ```bash
 clusterctl config cluster capi-quickstart --flavor development \
-  --kubernetes-version v1.18.16 \
+  --kubernetes-version v1.18.19 \
```

Comment:
Q: should we update to v1.18.19 for all the providers
@sbueringer opinions?

Reply:
In general sounds good to me. I guess we always have the problem that we don't know if they have published images. But that problem is the same for v1.18.16 and v1.18.19.

Reply:
Okay, I will change the version for other providers too, thanks @fabriziopandini and @sbueringer!